### ISSUE TITLE
FluentChoco won't work on latest HotChocolate v12

### ISSUE THREAD

**Issue by shidcordero** [dalrankov/FluentChoco#3]
here's the error it throws
```

message: "Method not found: '!!0 HotChocolate.Resolvers.IResolverContext.ArgumentValue(HotChocolate.NameString)'."
stackTrace: "   at FluentChoco.FluentValidationMiddleware.Invoke(IMiddlewareContext context)\r\n   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)\r\n   at FluentChoco.FluentValidationMiddleware.Invoke(IMiddlewareContext context)\r\n   at lambda_method7916(Closure , IMiddlewareContext , FluentValidationMiddleware )\r\n   at HotChocolate.Resolvers.FieldClassMiddlewareFactory.<>c__DisplayClass5_0`1.<CreateDelegate>b__1(IMiddlewareContext context)\r\n   at HotChocolate.AspNetCore.Authorization.AuthorizeMiddleware.InvokeAsync(IDirectiveContext context)\r\n   at HotChocolate.Utilities.MiddlewareCompiler`1.ExpressionHelper.AwaitTaskHelper(Task task)\r\n   at HotChocolate.Execution.Processing.Tasks.ResolverTask.ExecuteResolverPipelineAsync(CancellationToken cancellationToken)\r\n   at HotChocolate.Execution.Processing.Tasks.ResolverTask.TryExecuteAsync(CancellationToken cancellationToken)"

```

removing the `UseFluentValidation()` won't trigger that error but the fluent validation won't work anymore.

**Comment by dalrankov** [comment]
Hey @shidcordero, thanks for reporting this. I've released v3.0.0 that is compatible with HC v12+ and FV v10+.

**Comment by shidcordero** [comment]
@dalrankov it works great on HC 12.0. I just upgraded to 12.6 and it throws me this error 

```
message: "Method not found: 'HotChocolate.Types.IFieldCollection`1<HotChocolate.Types.IInputField> HotChocolate.Types.IOutputField.get_Arguments()'."
stackTrace: "   at FluentChoco.FluentValidationMiddleware.Invoke(IMiddlewareContext context)\r\n   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)\r\n   at FluentChoco.FluentValidationMiddleware.Invoke(IMiddlewareContext context)\r\n   at lambda_method4315(Closure , IMiddlewareContext , FluentValidationMiddleware )\r\n   at HotChocolate.Resolvers.FieldClassMiddlewareFactory.<>c__DisplayClass5_0`1.<CreateDelegate>b__1(IMiddlewareContext context)\r\n   at HotChocolate.AspNetCore.Authorization.AuthorizeMiddleware.InvokeAsync(IDirectiveContext context)\r\n   at HotChocolate.Utilities.MiddlewareCompiler`1.ExpressionHelper.AwaitTaskHelper(Task task)\r\n   at HotChocolate.Execution.Processing.Tasks.ResolverTask.ExecuteResolverPipelineAsync(CancellationToken cancellationToken)\r\n   at HotChocolate.Execution.Processing.Tasks.ResolverTask.TryExecuteAsync(CancellationToken cancellationToken)"

```
